lib/game_factory.rb

```diff
@@ -0,0 +1,26 @@
+class GameFactory
+
+  def initialize(player_factory, game_mode, display, board)
+    @player_factory = player_factory
+    @game_mode = game_mode
+    @display = display
+    @board = board
+  end
+
+  def create_game
+    welcome_players
+    present_board
+    players = @player_factory.set_up_players(@game_mode.choose_game_mode, @board, @display)
+    game = Game.new(@display, @board, players)
+  end
+
+  private
+
+  def welcome_players
```

Comment:
Is this the best class to call this from? Something feels a tad funny to me about having a method responsible for welcoming players in a GameFactory class :)